.github/workflows/ci.yml

```diff
@@ -314,6 +319,21 @@ jobs:
           cd sdk/tests/android/testapp/src/main/assets
           python3 -m pip install boto3
           python3 ./android_file_creation.py
+      
+      - name: Set Android keystore home
+        run: |
+          echo "ANDROID_SDK_HOME=$GITHUB_WORKSPACE/.android-home" >> "$GITHUB_ENV"
+          echo "ANDROID_PREFS_ROOT=$GITHUB_WORKSPACE/.android-home" >> "$GITHUB_ENV"
+          mkdir -p "$GITHUB_WORKSPACE/.android-home/.android"
+      - name: Create debug keystore
+        run: |
+          keytool -genkeypair \
```

Comment:
The keytool seems to be used to sign the android application. Is it a new thing for gradle 8.5.1?

Reply:
The validate sign debug part of updated gradle requires the application to be signed. Best guess is this is a new part of whatever the workflow is for building an app and for whatever reason we don't have write permission on github runners at the default location so we now just generate a key pair.